src/native-clipboard.coffee

```diff
@@ -0,0 +1,6 @@
+# Using clipboard in renderer process is not safe on Linux.
```

Comment:
What do you think about the name used here? To be honest, I don't like it much as it doesn't really express the intent of having a separate file. Maybe `safe-clipboard` may work better?